src/tests/ie_test_utils/common_test_utils/ngraph_test_utils.hpp

```diff
@@ -90,3 +90,17 @@ class TransformationTestsF : public  CommonTestUtils::TestsCommon {
     bool m_soft_names_comparison{true};
     bool m_enable_accuracy_check{false};
 };
+
+namespace {
+    void init_unique_names(std::shared_ptr<ngraph::Function> f, const std::shared_ptr<ngraph::pass::UniqueNamesHolder>& unh) {
```

Comment:
Please do not use anonymous namespaces in header files. https://rules.sonarsource.com/cpp/RSPEC-1000
Keep functions declaration in hpp and move definition to cpp.

Reply:
done